worker/src/main/java/gov/cms/ab2d/worker/processor/ContractProcessorImpl.java

```diff
@@ -114,9 +113,7 @@ public ContractProcessorImpl(JobRepository jobRepository,
 
             contractData.setStreamHelper(helper);
 
-            Iterator<Map.Entry<String, CoverageSummary>> patientEntries = patients.entrySet().iterator();
-
-            while (patientEntries.hasNext()) {
+            for (Map.Entry<String, CoverageSummary> stringCoverageSummaryEntry : patients.entrySet()) {
```

Comment:
Is this the best variable name? Each entry is a coverage summary for a beneficiary.